src/routes/Access.tsx

```diff
@@ -287,7 +287,7 @@ export function Access(props: IProps) {
       });
     }
     const userId = localStorage.getItem('online-user-id');
-    if (isElectron && userId && !curUser) {
+    if (isElectron && userId && !curUser && auth.isAuthenticated()) {
```

Comment:
I'm curious about this change. This doesn't seem to be part of this issue...

Reply:
ah, it was an early theory that it was going in here and getting lost because after we logout we come in here and it sets the user again which I don't think it should be doing.